src/platforms/javascript/index.mdx

```diff
@@ -6,7 +6,7 @@ categories:
   - browser
 ---
 
-On this page, we provide concise information to get you up and running with Sentry's JavaScript SDK, automatically reporting errors and exceptions in your application.
+On this page, we get you up and running with Sentry's JavaScript SDK, automatically reporting errors and exceptions in your application.
```

Comment:
```suggestion
On this page, we get you up and running with Sentry's JavaScript SDK, so that it will automatically report errors and exceptions in your application.
```